src/psl/psl-fsm.c

```diff
@@ -215,13 +215,11 @@ static fsm_state_t *build_sere(psl_fsm_t *fsm, fsm_state_t *state, psl_node_t p)
 {
    const int nops = psl_operands(p);
 
-   fsm_state_t *initial = state;
-
-   for (int i = 0; i < nops; i++) {
-      psl_node_t rhs = psl_operand(p, i);
+   for (int j = 0; j < nops; j++) {
```

Comment:
No need to change the variable name

Reply:
It was a left-over from previous implementation approach where this loop was nested within another loop.